src/measures.jl

```diff
@@ -83,6 +83,23 @@ function measure(t::Tetrahedron)
   abs((A - D) ⋅ ((B - D) × (C - D))) / 6
 end
 
+"""
+    measure(splx::Simplex)
+
+Compute the measure (i.e. hyper-volume) for a simplex in any dimension using the Cayley-Menger Determinant.
+"""
+function measure(splx::Simplex{K,Dim,T,N}) where {K,Dim,T<:Real,N}
+  # Find the equation e.g. at
+  # https://en.wikipedia.org/wiki/Cayley%E2%80%93Menger_determinant#Definition.
+  Ds_ = pairwise(SqEuclidean(), coordinates.(vertices(splx)))
```

Comment:
We don't use underscore in variable names.

Reply:
Done.